### ISSUE TITLE
Clarify label_weights parameter in tf.keras.metrics.AUC

### ISSUE THREAD

**Issue by jstremme** [tensorflow/tensorflow#48763]
### Summary of Documentation Issue in tf.keras.metrics.AUC

It is not clear whether or not the `label_weights` parameter in the [`tf.keras.metrics.AUC` documentation](https://www.tensorflow.org/api_docs/python/tf/keras/metrics/AUC) expects weights to  sum to 1 or not.  For example with multi-label data, you can take the sum of the occurrences of each label divided by the total number of samples to compute the following example weights:

![unscaled_label_weights](https://user-images.githubusercontent.com/17085758/116154273-13b3c180-a6ae-11eb-8f2a-242181de484f.png)

The weights above reflect the prevalence of each label.  You could also divide these weights by the sum of the weights array to put the weights in the range of [0, 1] like so:

![scaled_label_weights](https://user-images.githubusercontent.com/17085758/116154429-452c8d00-a6ae-11eb-8f6a-046856344dd8.png)

### Description of What to Clarify:

It would be helpful if the docs could be updated to clarify what these weights represent (the prevalence of each class, the scaled prevalence of each class on the range of [0, 1], or something else).  

If the docs seem straightforward to others and I am just missing something, please let me know.  I'm interested in computing micro-average PR AUC for my multi-label model, and it's not clear to me which of the following I should use for multi-hot encoded data.

##### Data Sample
```
>>> print(y_train)
>>> array([[1, 0, 1],
    [0, 1, 1],
    [0, 0, 0],
    [0, 1, 1]])
```

#### Unscaled Label Weights
```
n_samples = y_train.shape[0]
class_totals = y_train.sum(axis=0)
label_weights = class_totals / n_samples
```

#### Scaled Label Weights
```
n_samples = y_train.shape[0]
class_totals = y_train.sum(axis=0)
weights = class_totals / n_samples
label_weights = weights / sum(weights)
```

**Comment by jstremme** [comment]
I've been scaling the label weights which seems to be working well.  @fchollet, is this the intended approach?

**Comment by ymodak** [comment]
The docs are updated with the tf-nightly version.
See https://www.tensorflow.org/api_docs/python/tf/keras/metrics/AUC?version=nightly
